### ISSUE TITLE
TK3.3 - Bus operator micro journey

### ISSUE THREAD

**Issue by KaterinaKir** [wmcadigital/wmn-sprints#69]
**Epic:** What's the big picture?

***
#### As a
Commuter, leisure, business

#### I need to
understand which bus operator they use

#### So that
use knows if I need a single operator or multi-operator ticket

***
### Acceptance criteria
- [x] User understands that private operators run bus services, not WMN/TfWM/WMCA in plain English
- [x] User can search for a bus route by number
- [x] User is told which operator(s) run(s) the bus route

***

**Comment by daylesalmon** [comment]
Awaiting test and potential minor design change(s) based on test results.
Then re-assign to backlog/frontend.

**Comment by KaterinaKir** [comment]
Completed.
